curriculum/challenges/english/01-responsive-web-design/css-variables-skyline/part-004.md

```diff
@@ -7,47 +7,99 @@ dashedName: part-4
 
 # --description--
 
-Nest opening and closing `title` and `style` tags in the head area and give your project a title of `freeCodeCamp Skyline Project`. Any styles you are asked to add during this project should go in this style area.
+Add a `title` element to the `head`, and give your project a title of `freeCodeCamp Skyline Project`. Also, nest a self-closing `link` element in the `head` element. Give it a `rel` attribute value of `stylesheet`, a `type` attribute value of `text/css`, and an `href` attribute value of `styles.css`.
```

Comment:
```suggestion
Add a `title` element to the `head`, and give your project a title of `freeCodeCamp Skyline Project`. Also, nest a self-closing `link` element in the `head` element. Give it a `rel` attribute value of `stylesheet`, a `type` attribute value of `text/css`, and an `href` attribute value of `/styles.css`.
```
This could be side-effect of something else, but without `/` before the `styles.css` challenge will not pass, as `querySelectorAll` will not find added `link` tag.

Reply:
Instead of making this change, we should change the CSSHelp function, because both are valid methods to target the stylesheet (supposedly).

Reply:
That's not related to the CSSHelp, but to a change that was made to the way the iFrame renders the code I believe.

@ojeytonwilliams wrote something that removes the `link` and `script` tags for our "local files".

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/pull/42205#discussion_r643253252

Reply:
Right, I had forgotten that. Thank you